### PR TITLE
updates

### PR DESCRIPTION
**Pull request by jcstorms1**
### What does this PR do?

- Updates linux wrapper version
- Adds new information about Tomcat applications and that `DD_START_APP` is not necessary for them
- Removed unnecessary instructions in the Java section
### Motivation

New version released and support for Tomcat.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ This solution uses the startup command setting and Application Settings for Linu
...
+To instrument your application, begin by adding the following key-value pairs under **Application Settings** in your Azure configuration settings.
 
 {{< img src="serverless/azure_app_service/application-settings.jpg" alt="Azure App Service Configuration: the Application Settings, under the Configuration section of Settings in the Azure UI. Three settings are listed: DD_API_KEY, DD_SERVICE, and DD_START_APP." style="width:80%;" >}}
 
 - `DD_API_KEY` is your Datadog API key.
 - `DD_CUSTOM_METRICS_ENABLED` (optional) enables [custom metrics](#custom-metrics).
 - `DD_SITE` is the Datadog site [parameter][2]. Your site is {{< region-param key="dd_site" code="true" >}}. This value defaults to `datadoghq.com`.
 - `DD_SERVICE` is the service name used for this program. Defaults to the name field value in `package.json`.
-- `DD_START_APP` is the command used to start your application. For example, `node ./bin/www`.
+- `DD_START_APP` is the command used to start your application. For example, `node ./bin/www` (Unnecessary for applications running in Tomcat).
```
Comment (rtrieu) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:31]:
```suggestion
- `DD_START_APP` is the command used to start your application. For example, `node ./bin/www` (unnecessary for applications running in Tomcat).
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 2]
```diff
@@ -42,14 +42,14 @@ Set these values in the `DD_START_APP` environment variable. Examples below are
 | .NET Core | `dotnet datadog-demo.dll`                                                                  | Runs a `.dll` file that uses your Web App name by default. <br /><br /> **Note**: The `.dll` file name in the command should match the file name of your `.dll` file. In certain cases, this might not match your Web App.         |
 | PHP       | `cp /home/site/wwwroot/default /etc/nginx/sites-available/default && service nginx reload` | Copies script to correct location and starts applicatio.                                                                                                                                                                           |
```
Comment (rtrieu) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:43]:
```suggestion
| PHP       | `cp /home/site/wwwroot/default /etc/nginx/sites-available/default && service nginx reload` | Copies script to correct location and starts application.                                                                                                                                                                           |
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 3]
```diff
@@ -42,14 +42,14 @@ Set these values in the `DD_START_APP` environment variable. Examples below are
...
 | Python    | `gunicorn --bind=0.0.0.0 --timeout 600 quickstartproject.wsgi`                             | Custom [startup script][13]. This example shows a Gunicorn command for starting a Django app.                                                                                                                                      |
-| Java      | `java -jar /home/site/wwwroot/datadog-demo.jar`                                            | The command to start your app.                                                                                                                                                                                                     |
+| Java      | `java -jar /home/site/wwwroot/datadog-demo.jar`                                            | The command to start your app. Not required for applications running in tomcat                                                                                                                                                                                                    |
```
Comment (rtrieu) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:45]:
```suggestion
| Java      | `java -jar /home/site/wwwroot/datadog-demo.jar`                                            | The command to start your app. This is not required for applications running in Tomcat.                                                                                                                                                                                                  |
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor suggestions for consistency. Also found an existing typo.

**Comment by jcstorms1** [comment]
@rtrieu thanks for the suggestions. I commited them.

**Review by rtrieu — APPROVED** [review]
Looks good!

**Comment by jcstorms1** [comment]
@rtrieu Yes please!

**Pull request merged**